### ISSUE TITLE
error in seedTarget method

### ISSUE THREAD

**Issue by anand4234** [0x00string/oldays#9]
Hi, I tried you exploit for CUPS but when I try to run it I got an error:

```
python exp.py -a example.com -b 631 -c ./so.so

Traceback (most recent call last):
  File "exp.py", line 527, in <module>
    main()
  File "exp.py", line 516, in main
    jobid = seedTarget(rhost, rport, printer, payload)
  File "exp.py", line 220, in seedTarget
    "\x03"+
UnboundLocalError: local variable 'jobid' referenced before assignment
```

**Comment by 0x00string** [comment]
looks like maybe the target isn't letting you upload print jobs. its
failing during uploading copies of the payload.

**Comment by kyseif** [comment]
Hey @0x00string, would you be able to provide some additional insight? Might there be a CUPS setting that prevents uploading print jobs, or could it be something else? Other than /etc/cups/cupsd.conf, I'm not sure where else to look.

Also, I added some 'print' lines to the code and it looks like a job-id is being returned after running createJob, but I, too, receive the error, "local variable 'jobid' referenced before assignment"
![image](https://github.com/0x00string/oldays/assets/137201479/9e899926-8975-4a54-b121-168dc0183fa5)

**Comment by 0x00string** [comment]
nah, sorry. i wrote this i think over a decade ago. google p0 releaeed
their PoC for this around the same time i pushed this. i recommend using
that over this.
